lib/src/base-styles/_nx-tables.scss

```diff
@@ -365,29 +365,74 @@
   }
 }
 
-.nx-cell__sort-btn {
-  height: nx-table-variables.$table-cell-header-height;
+.nx-cell {
+  .nx-cell__sort-btn {
+    height: nx-table-variables.$table-cell-header-height;
+    position: relative;
+  
+    > * {
+      position: relative;
```

Comment:
Fairly sure this isn't needed.  The `.nx-cell__sort-btn` has three children:
1. the `::before`, which overrides this with `position: absolute`
2. A plain `span`, where I can't imagine the `position` mattering.
3. The `.nx-cell__sort-icons.fa-layers`, which already has `position: relative` set by `.fa-layers`.

Reply:
> In this case I was using the * selector, but it is to help with the stacking context for the span. Because the ::before has position: absolute applied (and if the span has no positioning), is disrupts the stacking context and the ::before ends up covering the span element.

Ok I'm able to reproduce that now.  I think a cleaner solution (in that it avoids a `*` selector and styling a bare `<span>`) would be to add `z-index: 0` to the `.nx-cell__sort-btn` in order to make it a stacking context, and then to add `z-index: -1` to the `::before` so that it takes its correct place behind everything else.

Reply:
> Ok I'm able to reproduce that now. I think a cleaner solution (in that it avoids a * selector and styling a bare <span>) would be to add z-index: 0 to the .nx-cell__sort-btn in order to make it a stacking context, and then to add z-index: -1 to the ::before so that it takes its correct place behind everything else.

Works and I agree, much cleaner: [ac13c94](https://github.com/sonatype/sonatype-react-shared-components/pull/1156/commits/ac13c94ee0d959ce2910a922fb9cb800e1a5b301)